src/core/spec/start/normalize/params/security.js

```diff
@@ -10,17 +10,40 @@ const IN_TO_LOCATION = require('./in_to_location')
 // Normalize OpenAPI security request parameters into specification-agnostic
 // format
 const getSecParams = function({
-  spec: { securityDefinitions, security: apiSecurity = [] },
+  spec: {
+    securityDefinitions,
+    security: apiSecurity = [],
+    components: {
+      securitySchemes: secSchemes
+    }
+  },
   operation: { security = apiSecurity },
 }) {
   const secRefs = getSecRefs({ security })
+  const secDef = getSecDefs({ securityDefinitions, securitySchemes: secSchemes})
   const secParams = secRefs.map(([secName, scopes]) =>
-    getSecParam({ secName, scopes, securityDefinitions }),
+    getSecParam({ secName, scopes, securityDefinitions: secDef }),
   )
   const secParamsA = Object.assign({}, ...secParams)
   return secParamsA
 }
 
+const getSecDefs = function({ securityDefinitions, securitySchemes }) {
```

Comment:
This isn't a good solution for checking 2.0 and 3.0 specs.  Just throwing up the most basic PR here.